Assets/_BForBoss/_Core/Scripts/Managers/RaceCourseWorldManager.cs

```diff
@@ -1,65 +0,0 @@
-using Perigon.Analytics;
-using Perigon.Leaderboard;
-using Perigon.Utility;
-using Sirenix.OdinInspector;
-using UnityEngine;
-using UnityEngine.Rendering;
-
-namespace BForBoss
-{
-    public class RaceCourseWorldManager : BaseWorldManager
```

Comment:
Is this really not being used by any of the scenes atm?

Reply:
it is being used in `MS1_RaceCourse.unity`

Reply:
Yeah I made the change in the other pr to only show the wave sceheu